### DISCUSSION TITLE
Tilt function

### DISCUSSION THREAD

**Discussion by aadkszcn** [aarons22/homebridge-bond#250] [General]
The Team at Bond, confirm that the Bond app will support both up/down, My and tilt open/close. On YouTube I have seen example of the up/down, but not of the tilt function. Will the tilt function also work with this plug-in?

**Comment by chrismerck** [comment]
Sorry, Bond Home does not support Tilt function. --- There may be very limited functionality through the app, but it is not exposed on the API and thus will not work with integrations. --- We hope to change this in the future.
